# ObjectNodeFromClass.srv

string object_name
---
int32 num_objects
ObjectNode[] objects
